### PR TITLE
abbr: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by sebastiaanspeck — APPROVED** [review]
LGTM!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Note to new maintainers**: Don't merge this PR until it is reviewed by the Code owner (native speaker).

File: pages.pl/linux/abbr.md
Comment (acuteenvy) [inline pages.pl/linux/abbr.md:21]:
```suggestion
> Zarządzaj skrótami dla powłoki fish.
> Zdefiniowane przez użytkownika słowa są zastępowane po wpisaniu dłuższymi zwrotami.
> Więcej informacji: <https://fishshell.com/docs/current/cmds/abbr.html>.

- Dodaj nowy skrót:

`abbr --add {{nazwa_skrótu}} {{polecenie}} {{argumenty_polecenia}}`

- Zmień nazwę istniejącego skrótu:

`abbr --rename {{stara_nazwa}} {{nowa_nazwa}}`

- Usuń istniejący skrót:

`abbr --erase {{nazwa_skrótu}}`

- Zaimportuj skróty zdefiniowane na innym hoście poprzez SSH:

`ssh {{nazwa_hosta}} abbr --show | source`
```

**Review by acuteenvy — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please use the imperative for descriptions, as per [the style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
